source/manual/conventions-for-rails-applications.html.md

```diff
@@ -423,4 +427,6 @@ approach][rspec-request-moj] to replace [controller specs][] reflecting a
 
 ### Useful reading
 
-[The RSpec Styleguide](https://rspec.rubystyle.guide/)
+[The RSpec Styleguide][]
+
+[The RSpec Styleguide]: https://rspec.rubystyle.guide
```

Comment:
This aspect of consistency seems a bit superfluous - arguably causes a wtf. No biggie though.